scripts/vsc-venv-wrapper/vsc-venv.sh

```diff
@@ -0,0 +1,97 @@
+SCRIPT_NAME=${BASH_SOURCE[0]} # $0 cannot be used as it gives '-bash' when sourced
+
+usage() {
+  echo "Usage: source $SCRIPT_NAME {activate <requirements.txt> [modules.sh] | deactivate}"
+  echo ""
+  echo "Commands:"
+  echo "  activate <requirements_file> [modules_script] Activate the environment using the specified requirements file."
+  echo "                                                Optionally, load modules from the specified script before activating the environment."
+  echo ""
+  echo "  deactivate                                    Deactivate the virtual environment."
+  echo ""
+  echo "Example Usage:"
+  echo "  $ source $SCRIPT_NAME activate requirements.txt modules.sh"
+  echo "  $ python my_script.py"
+  echo "  $ source $SCRIPT_NAME deactivate"
+  return 1
+}
+
+echo_info() { echo -e "\e[32m[INFO] $1\e[0m"; }
+echo_warning() { echo -e "\e[33m[WARNING] $1\e[0m"; }
+echo_error() { echo -e "\e[31m[ERROR] $1\e[0m"; }
+
+
+# ============================ Main functions ============================
+
+activate() {
+  REQUIREMENTS_FILE="$1"
```

Comment:
lowercase variables, and use `local ...` to make them scoped in the function
```bash
local requiremets_file  modules_script
requirements_file="$1"
...
```

Reply:
Fixed in [lower case local variables](https://github.com/hpcugent/vsc_user_docs/pull/670/commits/b228814481ce5cc861b3a4838562cc5b3fad6357)